support-frontend/assets/pages/[countryGroupId]/events/events.tsx

```diff
@@ -1,21 +1,64 @@
 import { css } from '@emotion/react';
-import { from, sport } from '@guardian/source/foundations';
-import { Container } from '@guardian/source/react-components';
+import { palette, space, textSans, until } from '@guardian/source/foundations';
+import { Column, Columns, Container } from '@guardian/source/react-components';
 import { FooterWithContents } from '@guardian/source-development-kitchen/react-components';
+import { Box, BoxContents } from 'components/checkoutBox/checkoutBox';
+import { CheckoutHeading } from 'components/checkoutHeading/checkoutHeading';
 import { Header } from 'components/headers/simpleHeader/simpleHeader';
 import { PageScaffold } from 'components/page/pageScaffold';
+import { guardianLiveTermsLink, privacyLink } from 'helpers/legal';
 import ThankYouFooter from 'pages/supporter-plus-thank-you/components/thankYouFooter';
 
-const checkoutContainer = css`
-	${from.tablet} {
-		background-color: ${sport[800]};
+const darkBackgroundContainerMobile = css`
+	background-color: ${palette.neutral[97]};
+	${until.tablet} {
+		background-color: ${palette.brand[400]};
+		border-bottom: 1px solid ${palette.brand[600]};
+	}
+`;
+
+const columns = css`
+	position: relative;
+	color: ${palette.neutral[7]};
+	${textSans.medium()};
+	padding-top: ${space[10]}px;
+	${until.tablet} {
+		padding-top: ${space[3]}px;
+	}
+`;
+
+const shorterBoxMargin = css`
+	border-radius: ${space[2]}px;
+	:not(:last-child) {
+		${until.tablet} {
+			margin-bottom: ${space[2]}px;
+		}
+	}
+`;
+
+const tscs = css`
+	color: #606060;
+	${textSans.xxsmall()};
+	padding-bottom: ${space[2]}px;
+	${until.tablet} {
+		color: ${palette.neutral[97]};
+	}
+	& a {
+		color: #606060;
+		${until.tablet} {
+			color: ${palette.neutral[97]};
+		}
 	}
 `;
 
 export function Events() {
 	const searchParams = new URLSearchParams(window.location.search);
 	const eventId = searchParams.get('eventId') ?? '4180362';
 	const chk = searchParams.get('chk') ?? '9fa2';
+
+	const termsEvents = <a href={guardianLiveTermsLink}>Terms and Conditions</a>;
+	const privacyPolicy = <a href={privacyLink}>Privacy Policy</a>;
```

Comment:
Should we make these their own components? Saying that, what's the benefit of having them abstracted like this?

Reply:
think as it stands its more for readability, think this approach has been done elsewhere. Own components perhaps a bit too much? Happy to change if you think otherwise?